configure

```diff
@@ -280,36 +280,33 @@ elif (args.eigen != ""):
 #
 if not args.comp in ['g++','icc','clang++']:
     raise Exception(color.red + "ERROR" + color.reset + ": Compiler must be g++, clang++, or icc")
-if (args.comp == "gcc"): f.write("COMP = GCC\n")
-elif (args.comp == "clang"): f.write("COMP = CLANG\n")
-elif (args.comp == "intel"): f.write("COMP = INTEL\n")
```

Comment:
This section should not be removed. The `COMP = ` lines are used to communicate compiler information to the metadata file.

Reply:
None of these conditionals are ever true, check the previous line.